src/main/java/ca/unknown/bot/use_cases/schedule_reminder/ScheduledReminderInputData.java

```diff
@@ -0,0 +1,60 @@
+package ca.unknown.bot.use_cases.schedule_reminder;
+
+import ca.unknown.bot.entities.schedule_reminder.EventDate;
+
+/**
+ * Converts user input data to a format that the interactor can use.
+ * @param eventName Stores the name of a scheduled event. This would refer to the course code for an Exam
+ *                  and the assignment name for an Assignment.
+ * @param eventDate Stores the date of a scheduled event in YY MM DD HR MIN SEC format. Refers to the
+ *                  exam date for an Exam and due date for an Assignment.
+ * @param location Stores the location of an Exam.
+ * @param assignmentCourseCode Stores the course code of an Assignment.
+ *
+ */
+public class ScheduledReminderInputData {
+    private String eventName;
+    private EventDate eventDate;
+    private String location;
+    private String assignmentCourseCode;
+
+    public ScheduledReminderInputData(String eventName, int[] eventDateNumeric){
+        this.eventName = eventName;
```

Comment:
Just curious, why subtract 1 from `eventDateNumeric[1]`?

Reply:
for some reason the month starts at 0 so inputting 7 will give you august